conf/test.config

```
@@ -22,7 +22,8 @@ params {
     input                       = "${projectDir}/assets/samplesheet.csv"
 
     // Blobtoolkit server
-    btk_location                = "GCA_927399515.1"
+    btk_location                = null
+    btk_online_location         = "https://blobtoolkit.genomehubs.org/api/v1/dataset/id/CALUEP01"
```

To remember the discrepancy is fine
```suggestion
    // Note: this doesn't match the main assembly being tested, but it doesn't matter
    btk_online_location         = "https://blobtoolkit.genomehubs.org/api/v1/dataset/id/CALUEP01"
```

---

That's fair, I must have left it in there from testing. Bad practise in it's own right i know.